tempto-ldap/src/main/java/com/teradata/tempto/fulfillment/ldap/LdapObjectDefinition.java

```diff
@@ -28,12 +29,14 @@
     private final String distinguishedName;
     private final Map<String, String> attributes;
     private final List<String> objectClasses;
+    private final Map<String, List<String>> modificationAttributes;
 
-    private LdapObjectDefinition(String id, String distinguishedName, Map<String, String> attributes, List<String> objectClasses)
+    private LdapObjectDefinition(String id, String distinguishedName, Map<String, String> attributes, Map<String, List<String>> modificationAttributes, List<String> objectClasses)
```

Comment:
I'd stick with `Map<..., List<...>>` in public API. Users can always create a multimap and do `Multimaps.asMap` if they prefer.